WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductCardPriceSummaryViewModel.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import WooFoundation
+
+/// View model for `CollapsibleProductCardPriceSummary`
+///
+struct CollapsibleProductCardPriceSummaryViewModel {
+    /// Whether the product is priced individually. Defaults to `true`.
+    ///
+    /// Used to control how the price is displayed, e.g. when a product is part of a bundle.
+    ///
+    let pricedIndividually: Bool
+
+    /// Unformatted product price
+    ///
+    private let price: String?
+
+    /// Quantity of product in the order. The source of truth is from the the quantity stepper view model `stepperViewModel`.
+    ///
+    private let quantity: Decimal
+
+    private let currencyFormatter: CurrencyFormatter
+
+    init(pricedIndividually: Bool,
+         quantity: Decimal,
+         price: String?,
+         currencyFormatter: CurrencyFormatter = CurrencyFormatter(currencySettings: ServiceLocator.currencySettings)) {
+        self.pricedIndividually = pricedIndividually
+        self.quantity = quantity
+        self.price = pricedIndividually ? price : "0"
```

Comment:
Just wondering, is the price logic for the variable product case still needed somewhere?

https://github.com/woocommerce/woocommerce-ios/blob/f2a47b8713371effa2320cfbbd2e271afdf74a56/WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductRowViewModel.swift#L294-L295

After this PR, we probably don't need `pricedIndividually` anymore since it's an order item-specific concept.

Reply:
Update: the variable product case isn't relevant in the order item context, because an order item for a variable product is always associated with a variation with a fixed price. Please ignore the question above!

Reply:
> After this PR, we probably don't need pricedIndividually anymore since it's an order item-specific concept.

We still need it for now, because `CollapsibleProductCardViewModel` is still setting the `pricedIndividually` value based on `ProductRowViewModel`:

https://github.com/woocommerce/woocommerce-ios/blob/76f1288ebca3d1c9a26d7993a775ff2fad21f984/WooCommerce/Classes/ViewRelated/Orders/Order%20Creation/CollapsibleProductRowCardViewModel.swift#L62

We're still mostly relying on `ProductRowViewModel` to provide the data for `CollapsibleProductCardViewModel`. But when I make the change in #11389 to rely on `OrderItem` data in `CollapsibleProductCardViewModel`, I should be able to remove the reliance on `ProductRowViewModel` and we can remove `pricedIndividually` from that view model at that point.